SD2019StackOverflowApp/src/test/java/ro/utcn/sd/mca/SD2019StackOverflowApp/IntegrationTests.java

```diff
@@ -0,0 +1,110 @@
+package ro.utcn.sd.mca.SD2019StackOverflowApp;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.entity.*;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.exception.InvalidAnswerIdException;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.exception.InvalidQuestionIdException;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.exception.InvalidSOUserIdException;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.persistence.RepositoryFactory;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.persistence.SpecificationFactory;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.service.AnswerManagementService;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.service.QuestionManagementService;
+import ro.utcn.sd.mca.SD2019StackOverflowApp.service.SOUserManagementService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("test")
+public class IntegrationTests {
+    @Autowired
+    private RepositoryFactory repositoryFactory;
+    @Autowired
+    private SpecificationFactory specificationFactory;
+    @Autowired
+    private SOUserManagementService sOUserService;
+    @Autowired
+    private QuestionManagementService questionService;
+    @Autowired
+    private AnswerManagementService answerService;
+
+    // instantiaza de 2 ori clasa asta nush de ce
+    private static boolean firstSeed = true;
```

Comment:
Comments in Romanian are bad! Always use English!

Also, `@Before` methods are called before **each test**. Use the `@BeforeAll` annotation to execute something before the whole class. See https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall